extension/src/js/background/index.js

```diff
@@ -64,21 +152,20 @@ function loadSocketIO() {
 }
 
 function injectAppScript() {
-    chrome.tabs.executeScript({
-        file: "js/inject.js"
-    }, function () {
-        if (chrome.runtime.lastError) {
-            console.error(chrome.runtime.lastError.message);
-        } else console.log("All scripts successfully loaded");
-    });
+    chrome.tabs.query({active: true, currentWindow: true}, function(tabs) {
+        var activeTab = tabs[0];
```

Comment:
We should probably match this with the current Tab ID. Lakshaya stores it in his PR, so maybe we can revisit this change and incorporate that.